cirq-core/cirq/contrib/graph_device/graph_device.py

```diff
@@ -125,15 +125,13 @@ def __init__(
         device_graph: Optional[UndirectedHypergraph] = None,
         crosstalk_graph: Optional[UndirectedHypergraph] = None,
     ) -> None:
```

Comment:
First line should be a description of the method. In this case I think the args should be moved to `Attributes` on the class. https://google.github.io/styleguide/pyguide.html#384-classes